### PR TITLE
Extra parameters

### PR DESCRIPTION
**Pull request by AndreaGiardini**
New parameters:
* app_name
* disable_registration
* require_signin_view

### CODE REVIEW

File: manifests/config.pp [hunk 1]
```diff
@@ -42,7 +46,7 @@
   }
 
   file { '/etc/gogs/conf/app.ini':
-    ensure  => 'file',
+    ensure  => $ini_ensure,
```
Comment (bjwschaap) [inline manifests/config.pp]: What is the advantage of making the ensure for the app.ini settable?? I'm not sure why you would want to do this?

**Comment by AndreaGiardini** [comment]
Sorry, forgot to delete that commit

Should be ok now

**Comment by AndreaGiardini** [comment]
Is it possible to push a new version including this change?

**Comment by AndreaGiardini** [comment]
I will submit another pull request before

**Pull request merged**